cypress-tests/cypress/support/commands/login.ts

```diff
@@ -28,7 +28,7 @@ const register = (
 
   // ? wait to be assess whether tokens were refreshed automatically or we really need to register
   cy.wait(1000); // eslint-disable-line cypress/no-unnecessary-waiting
-  cy.request({ failOnStatusCode: false, url: "ui-server/api/user" }).then(
+  cy.request({ failOnStatusCode: false, url: "ui-server/api/data/user" }).then(
```

Comment:
This points to the data service instead of the gateway. I would expect to authenticate with the gateway and check there if we are good to go. Has the authentication flow changed?

Reply:
The gateway has no endpoint to show user info or if a user is logged in or not. That request goes through the gateway where the credentials are added and then to the data service which has a database of all users and can return user information if presented with a valid token.

I dont think it really matters tbh where the request is served from. Going forward that will be the way to check if the user is logged in.

So I added this mostly for the future. Currenly the UI decides if the user is logged in by hitting Gitlab via the gateway. And that is the check I added right below.